client/src/pages/RoomPage.tsx

```diff
@@ -34,8 +34,8 @@ function RoomPage({ name }: { name: string }): JSX.Element {
   const { userInfos, addUserInfo, removeUserInfo } = useContext(UserInfoContext);
   const history = useHistory();
   const [showModal, setShowModal] = useState(false);
-
   const [network, setNetwork] = useState<Network>(null as unknown as Network);
```

Comment:
Since the reference depends on useState, the state is still needed.
See: https://stackoverflow.com/questions/62806541/how-to-solve-the-react-hook-closure-issue

Reply:
It doesn't seem like it needs to be a "state" of the component, since setState, which causes re-rendering is only called in useEffect. (It seem to work without the "state" network if you try it)